DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,11 +164,15 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
```

Comment:
@slava77 I'm not sure what you are asking. What was the actual failure you see? Is this from the exception containing the message
"Incorrect size read for the container in recoBeamHaloSummary_BeamHaloSummary__reRECO.obj.problematicStrips"
?